tensorflow/compiler/xla/service/gpu/gpu_conv_runner.cc

```diff
@@ -144,7 +144,55 @@ Status RunGpuConvForwardActivation(GpuConvParams params,
 // convolutions are supported here.
 template <typename ElementType, typename BiasType, typename OutputType,
           typename std::enable_if<
-              !std::is_integral<ElementType>::value>::type* = nullptr>
+              !std::is_integral<ElementType>::value &&
+              !std::is_same<ElementType, cus>::value>::type* = nullptr>
+Status RunGpuConvInternalImpl(GpuConvParams params,
```

Comment:
what is the difference between this RunGpuConvInternalImpl and the one below?

Reply:
The body of the function is the exact copy. So is it necessary to have two functions?

Reply:
it is the same copy. I copied it and thought I might need to change things for cus, turned out that I don't need to.